lib/auth/tun_test.go

```diff
@@ -138,7 +138,9 @@ func (s *TunSuite) TestUnixServerClient(c *C) {
 	otpSecret := base32.StdEncoding.EncodeToString([]byte(rawSecret))
 
 	user, role := createUserAndRole(s.a, userName, []string{userName})
-	role.SetResource(services.KindNode, services.RW())
+	rules := role.GetRules(services.Allow)
```

Comment:
I think that tests are first class user of the code, so I would say this is a vaild use case.